### PR TITLE
Add rest of the command to the driver

### PR DESCRIPTION
**Pull request by blackdwarf**
Currently, the driver just shows information about compile, publish and run. Add
info for init, pack and restore as well.

/cc @piotrpMSFT @brthor @anurse

### CODE REVIEW

**Comment by muratg** [comment]
How about `repl`?  Is that a thing now?

File: src/Microsoft.DotNet.Cli/Program.cs [hunk 1]
```diff
@@ -22,9 +22,12 @@ public class Program
...
 
 Common Commands:
+  init          Initialize a basic .NET project
+  restore       Restore dependencies specified in the .NET project
   compile       Compiles a .NET project
-  publish       Publishes a .NET project for deployment
-  run           Compiles and immediately executes a .NET project";
+  publish       Publishes a .NET project for deployment (including the runtime)
+  run           Compiles and immediately executes a .NET project
+  pack          Package a NuGet package of the application";
```
Comment (davidfowl) [inline src/Microsoft.DotNet.Cli/Program.cs]: This description doesn't make sense :smile:

Comment (blackdwarf) [inline src/Microsoft.DotNet.Cli/Program.cs]: It is a very meta thing. :) Do suggest an alternative.

**Comment by blackdwarf** [comment]
@muratg it is, but we are still discussing the actual name (see #182) so I want to include it later.

File: src/Microsoft.DotNet.Cli/Program.cs (hunk 1 above)
Comment (brthor) [inline src/Microsoft.DotNet.Cli/Program.cs]: `Create a NuGet package from the application` ?

The double package strikes me as off.

Comment (krwq) [inline src/Microsoft.DotNet.Cli/Program.cs]: Creates NuGet package or creates a package of the application (drop the word NuGet)

**Comment by brthor** [comment]
LGTM :shipit:

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please

#132 again

**Pull request merged**